Source/Plugin.BLE/Shared/Contracts/IDevice.cs

```diff
@@ -106,5 +106,18 @@ public interface IDevice : IDisposable
         /// <returns>True if the update request was sucessfull. On iOS it will always return false.</returns>
         /// <param name="interval">The requested interval (High/Low/Normal)</param>
         bool UpdateConnectionInterval(ConnectionInterval interval);
+
+
+        /// <summary>
+        /// Gets the information if the device has hinted during advertising that the device is connectable.
+        /// This information is not pat of an advertising record. It's determined from the PDU header.
+        /// Check CanIsConnectable to verify that the device supports IsConnectable.
+        /// </summary>
+        bool IsConnectable { get; }
+
+        /// <summary>
+        /// True, if device supports IsConnectable else False
+        /// </summary>
+        bool CanIsConnectable { get; }
```

Comment:
One rather 'cosmetic' comment here: `CanIsConnectable` does not seem to be a very good name to me (sounds a bit weird ;). How about calling it `SupportsIsConnectable` instead?

Reply:
Choosed Can... because of CanRead, CanWrite, CanUpdate of characteristics.
But Supports... is ok, I will change it.

Reply:
Well, yeah, for the other cases 'can' makes more sense grammatically, for 'IsConnectable' it really doesn't.
Thanks for your consideration. :)